src/App.tsx

```diff
@@ -1,24 +1,75 @@
 import React from 'react';
-import logo from './logo.svg';
+
 import './App.css';
 
+import Schedule from './components/Schedule';
+import { Layout } from 'antd';
+import moment from 'moment';
+import { IDailySchedule } from './components/DailySchedule';
+import Asana from './components/Asana';
+
+const { Sider, Content } = Layout
+
+const channels = {
+  default: {
+    title: 'default',
+    color: 'red'
+  },
+  personal: {
+    title: 'personal',
+    color: 'blue'
+  }
+}
+
 const App = () => {
+  const today = moment()
+  const schedules: IDailySchedule[] = [{
```

Comment:
**It honestly does not matter in this context**, but if this was a real/production feature I'd say:

Setting up this array here will make it so it's computed on every render, vs just once or initially (by moving that code outside, or into a `componentDidMount` or a `useEffect` hook), which can slow down performance with more tasks/data, so I'd suggest making that change.

Reply:
That is a very fair point. My plan is to move everything into a state once I had forms setup to change it. However I probably should have cleaned it up before pushing the PR. Thanks for catching it.

Reply:
That makes sense, thanks for clarifying! Like I said, it doesn’t matter much here, for the sake of this demo.